packages/rum-core/src/domain/view/viewMetrics/trackInteractionToNextPaint.spec.ts

```diff
@@ -52,7 +52,7 @@ describe('trackInteractionToNextPaint', () => {
     setViewEnd = interactionToNextPaintTracking.setViewEnd
 
     registerCleanupTask(() => {
-      interactionToNextPaintTracking.stop
+      interactionToNextPaintTracking.stop()
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏ nice catch!